getting-started/setting-env.md

```diff
@@ -1,11 +1,53 @@
 ---
-title: Cosmos SDK
 order: 3
 ---
 
-# Using Cosmos SDK
+# Setting Up Environment
 
-## Connect to Demo Net
+You need an environment to run contracts. You can either run your node locally or connect to an existing network.
+If you just want to test a sample contract, We recommend you to skip running the a local node and just use a [live testnet](https://github.com/CosmWasm/tesnets)
+
+## Setup wallet
+
+You need some cash for interacting with the network. Let's set it up:
+
+```sh
+# add more wallets for testing
+wasmcli keys add fred
+>
+{
+  "name": "fred",
+  "type": "local",
+  "address": "cosmos1avdvl5aje3zt0uay40uj6l9xtqtlqhduu84nql",
+  "pubkey": "cosmospub1addwnpepqvcjveqepq34x59fnmygdy58ag7zwu8gefgsprq9th38nxzptpgszc3rkve",
+  "mnemonic": "hobby bunker rotate piano satoshi planet network verify else market spring toward pledge turkey tip slim word jaguar congress thumb flag project chalk inspire"
+}
+wasmcli keys add bob
+wasmcli keys add thief
+```
+
+## Run Local Node (optional)
```

Comment:
I would move this to the bottom and just reference it here (default path just shows testnet, but a link "if you want to setup and run against a local blockchain, click here"